src/components/Markdown/Markdown.tsx

```diff
@@ -1,10 +1,22 @@
 import React from 'react'
 import remarkGfm from 'remark-gfm'
+import Image from 'next/image'
 import ReactMarkdown from 'react-markdown'
 import { ReactMarkdownOptions } from 'react-markdown/lib/react-markdown'
 
 const Markdown = (props: ReactMarkdownOptions) => {
-  return <ReactMarkdown remarkPlugins={[remarkGfm]} {...props} />
+  return (
+    <ReactMarkdown
+      components={{
+        img: ({ node, ...props }) => (
+          // eslint-disable-next-line @next/next/no-img-element
+          <img crossOrigin='anonymous' alt={props.alt} {...props} /> //https://stackoverflow.com/a/47359958/5837635
```

Comment:
Why shouldn't we use`next/image` here? The SO link seems just to relate to crossOrigin, which should be fine for `next/image` as well?

Reply:
Good question. It was simply because none of the props matched. E.g src, width etc were like `number | undefined` in props, and next/image expected e.g `string | FormattedXXX`. So I thought it would get so messy with lots of undefined checks and conversions, so I decided to go with <img>. The existing solution in prod probably parses this to native img:s already anyway.

Reply:
Fair?

Reply:
Alright, sounds fair!